services/scan/src/backup.ts

```diff
@@ -7,40 +7,21 @@ import Database from 'better-sqlite3';
 import { fileSync } from 'tmp';
 import ZipStream from 'zip-stream';
 import { FULL_LOG_PATH } from '@votingworks/logging';
-import { throwIllegalValue } from '@votingworks/utils';
 import { Store } from './store';
 
 const debug = makeDebug('scan:backup');
 
 /**
  * Specifies what to include in the backup
  *
- * @property scanImagesToInclude Specifies which scan images to include in the backup
+ * @property saveOnlyOriginalImagesThenOnlyNormalizedImagesAfterNumSheets If the number of sheets
+ * scanned by the machine is less than or equal to this, only original scan images will be included
+ * in the backup, and if the number of sheets scanned by the machine is greater, only normalized
+ * scan images will be included in the backup. If not specified, both original and normalized scan
+ * images will be included.
  */
 export interface BackupOptions {
-  scanImagesToInclude: 'all' | 'normalizedOnly' | 'originalOnly';
-}
-
-const defaultBackupOptions: BackupOptions = {
-  scanImagesToInclude: 'all',
-};
-
-/**
- * Parses a query param as a BackupOptions['scanImagesToInclude']
- */
-export function parseScanImagesToIncludeQueryParam(
-  param: qs.ParsedQs[string]
-): BackupOptions['scanImagesToInclude'] {
-  switch (param) {
-    case 'all':
-      return 'all';
-    case 'normalizedOnly':
-      return 'normalizedOnly';
-    case 'originalOnly':
-      return 'originalOnly';
-    default:
-      return defaultBackupOptions.scanImagesToInclude;
-  }
+  saveOnlyOriginalImagesThenOnlyNormalizedImagesAfterNumSheets?: number;
```

Comment:
Long variable name I know 😅 but the most accurate descriptor I could come up with